wp-content/plugins/ig-mpdf/vendor/mpdf/mpdf/.gitignore

```diff
@@ -0,0 +1,2 @@
+vendor/*
```

Comment:
to stay consistent, we should include this `vendor/`-directory in the repo

Reply:
This should be there, see changed files of this commit ...

Reply:
ok, pdf-generation works now.